### DISCUSSION TITLE
SwayNC

### DISCUSSION THREAD

**Discussion by isabelroses** [catppuccin/catppuccin#2009] [Port Requests]
### Will you make the port?

- [X] Yes, I'm willing to make the port

### How can the suggested port be themed?

https://github.com/ErikReider/SwayNotificationCenter#configuring

### Any additional comments?

I have alredy created the port: https://github.com/isabelroses/swaync

Previews:
https://raw.githubusercontent.com/isabelroses/swaync/main/assets/latte.webp
https://raw.githubusercontent.com/isabelroses/swaync/main/assets/frappe.webp
https://raw.githubusercontent.com/isabelroses/swaync/main/assets/macchiato.webp
https://raw.githubusercontent.com/isabelroses/swaync/main/assets/mocha.webp

### Submission Guidelines

- [X] I have read and followed Catppuccin's submission guidelines.

**Comment by vanillyn** [comment]
if youve already made the port, i believe you can make an issue on it

**Reply by isabelroses** [reply]
The issues are for port reviews I believe that's the stage after this one.

**Comment by isabelroses** [comment]
Closing for https://github.com/catppuccin/catppuccin/issues/2022
